development.ini

```diff
@@ -14,7 +14,13 @@ pyramid.default_locale_name = en
 pyramid.includes =
     pyramid_debugtoolbar
 
-sqlalchemy.url = sqlite:///%(here)s/learing_journal.sqlite
+# sqlalchemy.url = sqlite:///%(here)s/learing_journal.sqlite
+sqlalchemy.url = postgres://banksd:@localhost:5432/learing_journal
+# sqlalchemy.url = postgres://vlsxgagisvzhkx:h6yNw5aEYIPSyapUYzoZ1fkqMn@ec2-54-235-104-63.compute-1.amazonaws.com:5432/d8m6stckvjgohv
+# sqlalchemy.url = postgres://prflcwuvwuhtwz:rI5AYLSpblscpfStbpQG4Grbyy@ec2-54-221-201-244.compute-1.amazonaws.com:5432/dd8tcb4l3pm8cj
```

Comment:
golly, I hope these sqlalchemy urls are not the actual ones for your database, or I'll be able to connect to it directly myself.